pkg/steps/multi_stage.go

```diff
@@ -72,20 +69,18 @@ func MultiStageTestStep(
 	testConfig api.TestStepConfiguration,
 	config *api.ReleaseBuildConfiguration,
 	params api.Parameters,
-	podClient PodClient,
-	client ctrlruntimeclient.Client,
+	client PodClient,
 	artifactDir string,
 	jobSpec *api.JobSpec,
 ) api.Step {
-	return newMultiStageTestStep(testConfig, config, params, podClient, client, artifactDir, jobSpec)
+	return newMultiStageTestStep(testConfig, config, params, client, artifactDir, jobSpec)
 }
 
 func newMultiStageTestStep(
 	testConfig api.TestStepConfiguration,
 	config *api.ReleaseBuildConfiguration,
 	params api.Parameters,
-	podClient PodClient,
-	client ctrlruntimeclient.Client,
+	client PodClient,
```

Comment:
I don't understand this change - was the `ctrlruntimeclient` unused before?

Reply:
the `PodClient` is a superinterface of the `ctrlruntimeclient` and the pod extension interface now, so we can just use it where we used the `ctrlruntimeclient` before